src/pcap/cmd/pcap-bosh-cli/main.go

```diff
@@ -200,8 +202,11 @@ func checkOutputFile(file string, overwrite bool) error {
 	}
 	// File doesn't exist, check if path is valid
 	fileInfo, err := os.Stat(filepath.Dir(file))
-	if err != nil || !fileInfo.IsDir() {
-		return fmt.Errorf("cannot write file %s. %s does not exist", file, fileInfo.Name())
+	if err != nil {
+		if fileInfo != nil {
```

Comment:
Do we really need this additional check? It might be enough to return on error something like: 
`fmt.Errorf("cannot write file %s due to error: %w", file, err)`

Reply:
I wanted to retain the original behavior as much as possible, where the error message would print the directory that's missing. Before my change the code assumed `fileInfo` to never be nil which was clearly wrong.